lib/ansible/plugins/filter/ipaddr.py

```diff
@@ -90,11 +90,10 @@ def _6to4_query(v, vtype, value):
 
 
 def _ip_query(v):
-    if v.size == 1:
+    if v.size in [1, 2]:
         return str(v.ip)
     if v.size > 1:
```

Comment:
If the previous condition catches size=2, would this ever reach this condition for size==2?
```suggestion
     if v.size > 2:
```

Reply:
Good point, will update, thanks.